### ISSUE TITLE
GUI updater incorrectly shows dom0 update as succesful

### ISSUE THREAD

**Issue by unman** [QubesOS/qubes-issues#9364]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
4.2

### Brief summary
Dom0 update fails with error but summary shows Green Tick

### Steps to reproduce
GUI updater for dom0.
Updated packages are downloaded, but error:
'/usr/lib/qubes/qrexec-client-vm dom0 qubes.ReceiveUpdates /usr/lib/qubes/qfile-agent /var/lib/qubes/dom0-updates/packages/*.rpm' failed with exit code 1!

### Expected behavior
Summary will show error

### Actual behavior
Summary shows Green tick
Details report:
Updated packages: None

**Comment by marmarek** [comment]
I believe this just got fixed in https://github.com/QubesOS/qubes-desktop-linux-manager/pull/213

**Comment by alimirjamali** [comment]
> I believe this just got fixed in [QubesOS/qubes-desktop-linux-manager#213](https://github.com/QubesOS/qubes-desktop-linux-manager/pull/213)

Yes. It should fix it. Both for failed stage one (check for available updates) and failed stage 2 (actual updates e.g. because of a package conflict or network error).
